bigchaindb/tep/transactional_election.py

```diff
@@ -0,0 +1,38 @@
+
+class TransactionalElection(object):
+
+    version = '0.0'
+    name = 'base-class'
+
+    def __init__(self):
+        pass
+
+    def propose(self, election_object):
+        """Initiate a new election"""
+
+        raise NotImplementedError
+
+    def list(self, election_id):
```

Comment:
You did not implement it.. Do we really need this method? What do you think about calling it "get"?

Reply:
This method call will allow implementation for `bigchaindb upsert-validator show-request` which will be implemented in a seperate PR. @ttmc suggested that the sub-command should be called `show-request` so I think `show(self, election_id)` would be more appropriate?

Reply:
Yes, I like `show` better than `list`.